docs/source/examples.daisy_chain_gap.ipynb

```diff
@@ -0,0 +1,746 @@
+    "\n",
+    "  Documentation: https://rdkit.org/\n",
+    "\n",
+    "  Installation: `conda install -c conda-forge rdkit`\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "from pathlib import Path\n",
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "from ase.io import read, write\n",
+    "from ase import Atoms\n",
+    "\n",
+    "from xtb.ase.calculator import XTB\n",
+    "\n",
+    "from quippy.potential import Potential\n",
+    "\n",
+    "from wfl.configset import ConfigSet, OutputSpec\n",
+    "from wfl.generate import md\n",
+    "import wfl.descriptors.quippy\n",
+    "import wfl.select.by_descriptor\n",
+    "import wfl.fit.gap.simple\n",
+    "from wfl.calculators import generic\n",
+    "from wfl.autoparallelize.autoparainfo import AutoparaInfo\n",
+    "from wfl.autoparallelize.remoteinfo import RemoteInfo\n",
+    "from wfl.generate import smiles\n",
+    "from wfl.utils.configs import atomization_energy\n",
+    "from wfl.fit import error\n",
+    "from wfl.select.simple import by_bool_func\n",
+    "\n",
+    "from expyre.resources import Resources"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Reference calculator \n",
+    "\n",
+    "To parallelise (non-file-based) calculators, they need to be given to workflow functions as \n",
+    "\n",
+    "`(Initalizer, [args], {kwargs})`\n",
+    "\n",
+    "e.g. xtb would normally be called with \n",
+    "\n",
+    "`xtb_calc = XTB(method=\"GFN2-xTB\")`\n",
+    "\n",
+    "but instead in wfl scripts we define it as"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "xtb_calc = (XTB, [], {\"method\": \"GFN2-xTB\"})"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Prepare isolated atoms \n",
+    "\n",
+    "GAP requires reference (`e0`) energies for fitting. We construct `Atoms` objects with a single atom, evaluate them with the reference GFN2-xTB method and store in a file to later combine them with the training set. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "isolated_at_fname='isolated_atoms.xtb.xyz'\n",
+    "isolated_at = [Atoms(element, positions=[(0, 0, 0)]) for element in [\"H\", \"C\"]]\n",
+    "inputs = ConfigSet(isolated_at)\n",
+    "outputs = OutputSpec()\n",
+    "\n",
+    "inputs = generic.run(\n",
+    "    inputs=inputs,\n",
+    "    outputs=outputs,\n",
+    "    calculator=xtb_calc,\n",
+    "    properties=[\"energy\"],\n",
+    "    output_prefix=\"xtb_\"\n",
+    ")\n",
+    "\n",
+    "outputs = OutputSpec(isolated_at_fname)\n",
```

Comment:
I'm not sure that we should be encouraging users to deal with this kind of OutputSpec.store() loop routinely.  Honestly, what _I_ would probably like is acknowledge the fact that the calculation result is in memory, modify the Atoms objects in memory, and then write them to a file.  Would it be helpful to have a `ConfigSet.write(filename(s))` method?

Reply:
Should we directly pass one or more filenames, or should we pass an outputspec, so it could be used like `ConfigSet.write(OutputSpec(filename(s)))` ?

Reply:
Or should it actually be an `OutputSpec.write(ConfigSet)` method?  You could use it like
```
OutputSpec("filename.xyz").write(configset_of_stuff)
```

Reply:
I think the latter makes a bit more sense. 

While I think this would still be useful, I think this change is less crucial now that I've discovered `wfl.map`

Reply:
If c5a63de looks OK, can you resolve this conversation?

Reply:
I also re-added `OutputSpec(... set_tags=dict)` in 5445057423f

Reply:
Otherwise looks great, thanks!

Reply:
Sounds fine to me.  Can you do that change, and resolve this conversation?